CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
```

Comment:
can you remove this? changelog is managed automatically, and releases go out (nearly) every PR :sweat_smile: <3

Reply:
instead, could you add to the readme? there is already a subtle section that information about how to use `reaction` can go under :tada:

Reply:
done